### DISCUSSION TITLE
warning on you should not extend "group:" presets

### DISCUSSION THREAD

**Discussion by valdisrigdon** [renovatebot/renovate#26728] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.134.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### What is your question?

I'm getting a warning `you should not extend "group:" presets`. I want to disable updates for this group, so I've tried:

```
{
      "extends": "group:kubernetes",
      "enabled": false
}
```

What's the correct way to disable or change a packageRule for a group?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
`ignorePresets`

**Reply by valdisrigdon** [reply]
if I set `ignorePresets: ["group:kubernetes"]`, won't that just ignore the *grouping* but `gomod` manager will still update those?

**Reply by rarkins** [reply, marked as answer]
You can see the definition of group:kubernetes here: https://docs.renovatebot.com/presets-group/#groupkubernetes

So you can duplicate that matchPackagePrefixes or probably make a much smaller/more general rule like matchPackagePrefixes=k8s.io/
